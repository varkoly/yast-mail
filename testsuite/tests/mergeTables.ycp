/**
 * Module:
 *   NIS client configuration
 *
 * Summary:
 *   Testsuite
 *
 * Authors:
 *   Martin Vidner <mvidner@suse.cz>
 *
 * $Id$
 */
{
    include "testsuite.ycp";
    import "Mail";

    list(map) old1 = [
$["comment":" Basic system aliases that MUST be present.
","value":"root","key":"postmaster",] ,
$["comment":"","value":"postmaster","key":"mailer-daemon",] ,
$["comment":" amavis
","value":"root","key":"virusalert",] ,
$["comment":" General redirections for pseudo accounts in /etc/passwd.
","value":"root","key":"administrator",] ,
$["comment":"","value":"root","key":"daemon",] ,
$["comment":"","value":"root","key":"nobody",] ,
$["comment":" \"bin\" used to be in /etc/passwd
","value":"root","key":"bin",] ,
$["comment":" Further well-known aliases for dns/news/ftp/mail/fax/web/gnats.
","value":"news","key":"newsadm",] ,
$["comment":"","value":"news","key":"newsadmin",] ,
$["comment":"","value":"news","key":"usenet",] ,
];
    list(map) new1 = [
	$[ "key": "root", "value": "auser", "comment": " I am r00t!",],
	$[ "key": "B.User", "value": "buser", "comment": " blah",],
	$[ "key": "usenet", "value": "root", "comment": " direct",],
	$[ "key": "A.User", "value": "auser", "comment": "",],
	$[ "key": "newsadmin", "value": "root", "comment": " direct",],	
	];

    TEST (``(Mail::mergeTables ([], [])), [], nil);
    TEST (``(Mail::mergeTables (new1, [])), [], nil);
    TEST (``(Mail::mergeTables ([], old1)), [], nil);
    TEST (``(Mail::mergeTables (new1, old1)), [], nil);
}
