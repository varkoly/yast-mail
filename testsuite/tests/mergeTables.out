Read	.target.size "/etc/install.inf" nil
Read	.etc.install_inf.Sourcemounted nil
Read	.etc.install_inf.AutoYaST nil
Read	.etc.install_inf.Repair nil
Read	.etc.install_inf.Demo nil
Read	.etc.install_inf.Textmode nil
Read	.etc.install_inf.Manual nil
Read	.target.string "/proc/cmdline" nil
Read	.etc.install_inf.Console nil
Read	.etc.install_inf.Braille nil
Read	.etc.install_inf.VNC nil
Read	.etc.install_inf.UseSSH nil
Read	.etc.install_inf.Locale nil
Read	.etc.install_inf.InstMode nil
Return	[]
Return	[$["comment":" I am r00t!", "key":"root", "value":"auser"], $["comment":" blah", "key":"B.User", "value":"buser"], $["comment":" direct", "key":"usenet", "value":"root"], $["comment":"", "key":"A.User", "value":"auser"], $["comment":" direct", "key":"newsadmin", "value":"root"]]
Return	[$["comment":" Basic system aliases that MUST be present.\n", "key":"postmaster", "value":"root"], $["comment":"", "key":"mailer-daemon", "value":"postmaster"], $["comment":" amavis\n", "key":"virusalert", "value":"root"], $["comment":" General redirections for pseudo accounts in /etc/passwd.\n", "key":"administrator", "value":"root"], $["comment":"", "key":"daemon", "value":"root"], $["comment":"", "key":"nobody", "value":"root"], $["comment":" \"bin\" used to be in /etc/passwd\n", "key":"bin", "value":"root"], $["comment":" Further well-known aliases for dns/news/ftp/mail/fax/web/gnats.\n", "key":"newsadm", "value":"news"], $["comment":"", "key":"newsadmin", "value":"news"], $["comment":"", "key":"usenet", "value":"news"]]
Return	[$["comment":" Basic system aliases that MUST be present.\n", "key":"postmaster", "value":"root"], $["comment":"", "key":"mailer-daemon", "value":"postmaster"], $["comment":" amavis\n", "key":"virusalert", "value":"root"], $["comment":" General redirections for pseudo accounts in /etc/passwd.\n", "key":"administrator", "value":"root"], $["comment":"", "key":"daemon", "value":"root"], $["comment":"", "key":"nobody", "value":"root"], $["comment":" \"bin\" used to be in /etc/passwd\n", "key":"bin", "value":"root"], $["comment":" Further well-known aliases for dns/news/ftp/mail/fax/web/gnats.\n", "key":"newsadm", "value":"news"], $["comment":" direct", "key":"newsadmin", "value":"root"], $["comment":" direct", "key":"usenet", "value":"root"], $["comment":" I am r00t!", "key":"root", "value":"auser"], $["comment":" blah", "key":"B.User", "value":"buser"], $["comment":"", "key":"A.User", "value":"auser"]]
